Sources/MapboxNavigation/NavigationViewLayout.swift

```diff
@@ -281,10 +281,4 @@ extension NavigationView {
         compactConstraints.forEach({ $0.isActive = traitCollection.verticalSizeClass == .compact })
         regularConstraints.forEach({ $0.isActive = traitCollection.verticalSizeClass == .regular })
     }
-    
-    open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        if previousTraitCollection == traitCollection { return }
-        
-        setupConstraints()
```

Comment:
Why you've decided to call this method only during orientation change?

Reply:
Because on mobile the size class changes when the device rotates and a duplicate call for the orientation change. But it happens on other circumstances as well on other devices. I'll add it back.